salt/auth/rest.py

```diff
@@ -31,12 +31,14 @@
 
 __virtualname__ = "rest"
 
+cached_acl = {}
```

Comment:
Why do we need to maintain a cache?

Reply:
I'm not the original writer of the PR, I think it's mainly to reduce the number of API call. I can remove it.

Reply:
I prefer the caching part gets removed.

Reply:
Ok, I've removed the cache part.